apps/server/src/modules/learnroom/controller/dto/roomBoardResponse.ts

```diff
@@ -0,0 +1,95 @@
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import { DecodeHtmlEntities } from '@shared/controller';
+import { BoardTaskStatusResponse } from './roomBoardResponse-taskStatus';
+
+export class BoardTaskResponse {
+	constructor({ id, name, createdAt, updatedAt, status }: BoardTaskResponse) {
+		this.id = id;
+		this.name = name;
+		this.createdAt = createdAt;
+		this.updatedAt = updatedAt;
+		this.status = status;
+	}
+
+	@ApiProperty()
+	id: string;
+
+	@ApiProperty()
+	@DecodeHtmlEntities()
+	name: string;
+
+	@ApiPropertyOptional()
+	availableDate?: Date;
+
+	@ApiPropertyOptional()
+	duedate?: Date;
```

Comment:
this should be also renamed to "dueDate". I just saw this mistake within the frontend